docs/how-content-is-written.md

```diff
@@ -0,0 +1,36 @@
+# How content is written
+Content is written using [Markdown](https://guides.github.com/features/mastering-markdown/).
+
+## Guide to frontmatter
+
+### Valid frontmatter properties
+- [`layout`](#layout)
+- [`template`](#template)
+- [`title`](#title)
+- [`display_title`](#display_title)
+- [`description`](#description)
+- [`plainlanguage`](#plainlanguage)
+- [`collection`](#collection)
+- [`spoke`](#spoke)
+- [`order`](#order)
+- [`lastupdate`](#lastupdate)
+
+#### layout
+
+#### template
+
+#### title
+
+#### display_title
+
+#### description
+
+#### plainlanguage
+
+#### collection
+
+#### spoke
+
+#### order
+
+#### lastupdate
```

Comment:
@aubreyarcangel Feel free to update the format to your preference, but as you explore the source code please document any frontmatter properties here as you discover them!